samples/operate-first/filterClusterScoped.py

```diff
@@ -10,16 +10,40 @@ def splitToYamls(yamlFile):
             if yml:
                 filename = yml['metadata']['name']
                 outputFolder = yml['kind'].lower()+'s'
-                outputPath = os.path.join(outputFolder, filename + ".yaml")
+                outputPath = os.path.join(outputFolder, 'm4d-system', filename + ".yaml")
                 if not os.path.exists(os.path.dirname(outputPath)):
                     os.mkdir(os.path.dirname(outputPath))
```

Comment:
Since this is now a nested folder please change to use os.makedirs with exit_ok=True instead of using mkdir. You then also don't need the os.path.exists check